addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -589,11 +598,33 @@ registry.Parallax = Animation.extend({
 
         // Provide a "safe-area" to limit parallax
         const absoluteRatio = Math.abs(this.ratio);
-        this.$bg.css({
+        this._updateBgCss({
             top: -absoluteRatio,
             bottom: -absoluteRatio,
         });
     },
+    /**
+     * Updates the parallax background element style with the provided CSS
+     * values.
+     * If the editor is enabled, it deactivates the observer during the CSS
+     * update.
+     *
+     * @param {Object} cssValues - The CSS values to apply to the background.
+     */
+    _updateBgCss(cssValues) {
+        if (!this.$bg.length) {
+            // Safety net in case the `destroy` is called before the `start` is
+            // executed.
+            return;
+        }
```

Comment:
Indeed, my bad. I changed it.
Note that there is at least 5-10 occurrences in the code (in 15) where the same issue could arise in website and web_editor: doing some operation on this.xx on destroy where this.xx is set in start, and where xx is not `$el` or `$target`.
Should we fix those too?

Reply:
Of course, not critical though I guess. In case of public widgets, the main flows starting them and destroying them are ***normally*** correctly sequential. But it's a recurring issue that widgets in general crash on destroy because doing things before the start was even called, so yes probably should be fixed to be sure.

Reply:
Created a followup task: https://www.odoo.com/web#menu_id=4720&cids=1&action=333&active_id=974&model=project.task&view_type=form&id=3682498
About `this.$el` and `this.$target`, those are not a problem, right?

Reply:
No, unless you don't even try to put them in the DOM / attach them to a DOM before destroying them I guess. We don't care about that.